test/scenarios/create_dir_with_utf8_character/remote/changes.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "_id": "5a4845b78561d1c5970243cbe40012a5",
+    "_rev": "1-58193d0119d3f773f8982101206b5dc6",
+    "created_at": "2019-07-26T14:04:13.704599113Z",
+    "dir_id": "io.cozy.files.root-dir",
+    "name": "Partages reçus",
+    "path": "/Partages reçus",
```

Comment:
The name and path are encoded using NFC as this seems to be the encoding used in SWIFT.